ci-operator/step-registry/redhat-performance/benchmark-runner/redhat-performance-benchmark-runner-ref.yaml 

```diff
@@ -0,0 +1,7 @@
+  ref:
```

Comment:
I think if you run "make update" this will create another ref file ending in *-ref.metadata.json 
think this missing file is part of why your checks are failing